app/views/idv/doc_auth/send_link.html.erb

```diff
@@ -27,9 +27,6 @@
         delivery_methods: [:sms],
         class: 'margin-bottom-4',
       ) %>
-  <div class="usa-error-message margin-bottom-4 display-none" role="alert"></div>
```

Comment:
I can't see that this alert is actually referenced by anything. The `PhoneInputComponent` above renders its own error message through its inner `ValidatedFieldComponent`.